mantidproject/mantid
# Improved docs first index page

**Description of work.**

This has been adjusted to be an improvement to the first index page on loading the documentation, to make it more user focussed and clearer where novice users can go for help.


**To test:**

* Build the `docs-html` target of Mantid. I found that on OSX I had to build a regular target first then build the `docs-html` target
* Check the front index page of documentation 
* Check that the text makes sense

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
